tutor/src/screens/my-courses/new-teacher.tsx

```diff
@@ -166,6 +166,17 @@ const Content = styled.div`
             }
         }
     }
+
+    .ap-disclaimer {
+        width: 592px;
```

Comment:
why do we need to set the width to such a strange value?  Are we attempting to cause the para to break or something? If so, it'd be better to add 2 paras IMO

Reply:
that is the width that is set in the offering containers.